support/cas-server-support-pm-webflow/src/test/java/org/apereo/cas/pm/web/flow/actions/ValidatePasswordResetTokenActionTests.java

```diff
@@ -31,6 +39,24 @@ void verifyInvalidTicket() throws Throwable {
         assertEquals(CasWebflowConstants.TRANSITION_ID_INVALID_PASSWORD_RESET_TOKEN, validatePasswordResetTokenAction.execute(context).getId());
     }
 
+    @Test
+    void verifyGoodTicket() throws Throwable {
+        ClientInfoHolder.setClientInfo(new ClientInfo());
+        val id = UUID.randomUUID().toString();
+        val ticket = new TransientSessionTicketImpl();
```

Comment:
You like should switch to using a proper ctor:

```java
 val ticket = new TransientSessionTicketImpl("TST-1", NeverExpiresExpirationPolicy.INSTANCE,
            CoreAuthenticationTestUtils.getService(), Map.of("key", "value"));
```

Reply:
Will do.